autoload/SpaceVim/autocmds.vim

```diff
@@ -159,6 +159,7 @@ function! SpaceVim#autocmds#VimEnter() abort
     call SpaceVim#layers#core#tabline#def_colors()
     set showtabline=2
   endif
+  call SpaceVim#plugins#projectmanager#RootchandgeCallback()
```

Comment:
s/chandge/changed/ ?

Reply:
sorry，it has been merged， But I will fix this type, thank your for reviewing code.